
#  The checkAuth lambda/eval mess has a way of trying to use the last checkAuth in
#  for everything (or everything in that model.py), make sure there is a test to guard
#  aginst that


def test_django_stuff():
  pass
